flyteadmin/pkg/runtime/interfaces/application_configuration.go

```diff
@@ -50,6 +50,7 @@ type PostgresConfig struct {
 
 type FeatureGates struct {
 	EnableArtifacts bool `json:"enableArtifacts" pflag:",Enable artifacts feature."`
+	EnableHumanHash bool `json:"enableHumanHash" pflag:",Enable humanhash feature."`
```

Comment:
Can we rename this to `EnableFriendlyNames`?

```suggestion
	EnableFriendlyNames bool `json:"enableFriendlyNames" pflag:",Enable generation of friendly execution names feature."`
```